apps/files/src/components/FileList.vue

```diff
@@ -302,11 +300,17 @@ export default {
 
     actionsDropdownButtonId (id, active) {
       if (active) {
-        return `files-file-list-action-button-small-resolution-${id}-active`
+        return `files-file-list-action-button-${id}-active`
       }
 
-      return `files-file-list-action-button-small-resolution-${id}`
+      return `files-file-list-action-button-${id}`
     }
   }
 }
 </script>
+<style scoped>
+  /* FIXME: move to ODS, it is useful to have a column as wide as just an icon */
```

Comment:
When you are raising a ticket for this, add icon size classes as an idea. So `oc-icon-width` with the default icon width, then `oc-icon-width-xsmall`, `oc-icon-width-medium`, etc.